service/telemetry.go

```diff
@@ -123,6 +136,19 @@ func (tel *telemetryInitializer) initOnce(buildInfo component.BuildInfo, logger
 		telAttrs[semconv.AttributeServiceVersion] = buildInfo.Version
 	}
 
+	if tel.registry.IsEnabled(allowTraceContextPropagationFeatureGateID) {
+		props := strings.Split(cfg.Traces.Propagators, ",")
+
+		textMapPropagators, err := autoprop.TextMapPropagator(props...)
```

Comment:
This feature requires the use of OpenTelemetry SDK, which is part of another feature gate, right?

Reply:
Actually no, the feature is merged in master (not yet released) - no feature gates here.

Reply:
@jpkrohling  - I will move the PR from draft when go contrib officially releases the feature - Docs will be included - Waiting for the release to happen.

Reply:
@jpkrohling Actually no, that feature gate is for using OpenTelemetry SDK metrics (only) - OpenTelemetry SDK was always used for tracing without any feature flags.